### DISCUSSION TITLE
Sync new machine while one full node is running (same wallet) ?

### DISCUSSION THREAD

**Discussion by Pikislavcheck** [Chia-Network/chia-blockchain#8210] [Q&A]
Yo!

I have one full node farmer PC and two plotters/harvesters.
I would like to make another full node farmer and plug all plotted disks to it so that I will have 3 free PC for plotting/harvesting and this late one only for farming with full node with the same wallet.
I already installed Chia Gui on new PC but it doesnt want to sync.  I disabled upnp on this new one because I am runing synced full node with original 3PC setup that is working right now. 
Is it possible to sync another full node (gui,win) while "the old one" is still running ? 
I am asking this because I dont want to shut down "the old one" and wait a day or two for new one to sync.
I have open port, static IP and very good internet connection,

Thanks

**Comment by BrandtH22** [comment]
@Pikislavcheck , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note for multi-device farming it is recommended to review and follow the setup instructions here (only one full node is needed): https://docs.chia.net/farming-on-many-machines/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
